Cargo.toml

```diff
@@ -635,7 +635,7 @@ sinks-aws_cloudwatch_logs = ["aws-core", "aws-sdk-cloudwatchlogs"]
 sinks-aws_cloudwatch_metrics = ["aws-core", "aws-sdk-cloudwatch"]
 sinks-aws_kinesis_firehose = ["aws-core", "aws-sdk-firehose"]
 sinks-aws_kinesis_streams = ["aws-core", "aws-sdk-kinesis"]
-sinks-aws_s3 = ["base64", "md-5", "aws-core", "aws-sdk-s3"]
+sinks-aws_s3 = ["base64", "md-5", "aws-core", "aws-sdk-s3", "codecs"]
```

Comment:
If this is eventually going to be everywhere, are we getting much value out of the feature flag?

Reply:
True that, added an issue for this: https://github.com/vectordotdev/vector/issues/12160